src/main/java/mekanism/common/integration/OreDictManager.java

```diff
@@ -348,4 +393,35 @@ public static void addStandardOredictMetal(String suffix) {
             }
         }
     }
+    
+    public static void addStandardOredictGem(String suffix) {
+        NonNullList<ItemStack> gems = OreDictionary.getOres("gem" + suffix);
+        NonNullList<ItemStack> dusts = OreDictionary.getOres("dust" + suffix);
+        NonNullList<ItemStack> ores = OreDictionary.getOres("ore" + suffix);
+        for (ItemStack ore : ores) {
+            RecipeHandler.addEnrichmentChamberRecipe(StackUtils.size(ore, 1), StackUtils.size(gems.get(0), 2));
+        }
+
+        if (dusts.size() > 0) {
+            for (ItemStack ore : gems) {
```

Comment:
please fix your `for` loop variables

Reply:
@thiakil what is wrong with this for loop?
do you mean it shouldn't be name ore? but the entire class does it like that.

Reply:
Indeed, something looping on `gems` should call it's variable `gem`

I know there's lots of legacy cruft, but new code should do better - fixing the old ones is optional

Reply:
ok i will change that after i finished some testing with andother mod.